cidr2ip/main.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/fatih/color"
+	"github.com/ipinfo/cli/lib"
+	"github.com/ipinfo/cli/lib/complete/install"
+	"github.com/spf13/pflag"
+)
+
+var progBase = filepath.Base(os.Args[0])
+var version = "1.0.0"
+
+func printHelp() {
+	fmt.Printf(
+		`Usage: %s [<opts>] <cidrs | filepath>
+
+Description:
+	Accepts CIDRs and file paths to files containing CIDRs, converting
```

Comment:
```suggestion
  Accepts CIDRs and file paths to files containing CIDRs, converting
```

For help message text, don't use the tab character `\t`, otherwise it shows up incorrectly for users. Please only use raw spaces.

Reply:
And please double-check if the same thing was done for `range2ip` help messages or not.